packages/outputarea/src/widget.ts

```diff
@@ -1559,16 +1559,15 @@ namespace Private {
       const msg = 'Show more outputs';
       node.insertAdjacentHTML(
         'afterbegin',
-        `<a title=${title}>
-          <pre>${msg}</pre>
-        </a>`
+        `<button type="button" title="${title}" class="jp-TrimmedOutputs-button">
```

Comment:
Can you use `document.createElement('button')` to avoid security pitfals?

Reply:
Yes absolutely, was just looking into that.